pkg/handlers/generic/lifecycle/csi/handler_test.go

```diff
@@ -1,66 +0,0 @@
-// Copyright 2023 D2iQ, Inc. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package csi
-
-import (
-	"strings"
-	"testing"
-
-	"github.com/go-logr/logr"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
-var startAWSConfigMap = `
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-  name: ebs-sc
-parameters:
-  csi.storage.k8s.io/fstype: ext4
-  type: gp3
-provisioner: ebs.csi.aws.com
-volumeBindingMode: WaitForFirstConsumer
-`
-
-func Test_setDefaultStorageClass(t *testing.T) {
```

Comment:
Are there any tests we want to keep from here?

Reply:
unfortunately not. a lot of the stuff has been refactored into utils